onnxruntime/core/providers/op_kernel_type_control.h

```diff
@@ -146,6 +169,16 @@ struct EnabledTypes {
 
 // public macros
 
+/** Data types that are used in DataTypeImpl::AllTensorTypes()
+*/
+#define ORT_OP_KERNEL_TYPE_CTRL_ALL_TENSOR_DATA_TYPES \
+  bool,                                               \
```

Comment:
> bool,   [](start = 1, length = 8)

I don't know if the order of declaration should reflect the frequency of use. If so, we may want to move this down close to the string.

Reply:
shouldn't matter as this is used at compile time. 
the order in the kernel implementation can matter when a switch statement is involve, but doesn't matter if a type dispatcher is used as we iterate all types in the list.

---
In reply to: [569688210](https://github.com/microsoft/onnxruntime/pull/6547#discussion_r569688210) [](ancestors = 569688210)